scripts/explain_manifest/fixtures/amazonlinux-2-amd64.txt

```diff
@@ -200,6 +200,7 @@
   - lua-kong-nginx-module
   - lua-kong-nginx-module/stream
   - lua-resty-events
+  - ngx_brotli
```

Comment:
think putting it after the line of `lua-resty-lmdb` looks better as you did in other fixtures/*.txt

Reply:
done